docs/admin/runai-setup/notifications/notifications.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Notifications
+summary: This article describes the notifications that are available to the Run:ai platform, and how to configure them.
+authors:
+    - Jason Novich
+date: 2024-Jul-4
+---
+
+## Email Notifications
+
+This document provides an overview of Run:ai's Email Notifications feature, designed to keep data scientists informed about their workload statuses.
+
+### Importance of Email Notifications for Data Scientists
+
+Managing numerous data science workloads requires monitoring various stages, including submission, scheduling, initialization, execution, and completion. Additionally, handling suspensions and failures is crucial for ensuring timely workload completion.
+Email Notifications, introduced in Run:ai version 2.18, address this need by sending alerts for critical workload life cycle changes. This empowers data scientists to take necessary actions and prevent delays.
+
+### Setting Up Email Notifications
+
+The system administrator needs to enable and setup email notifications so that users are kept informed about different system statuses.
+
+To enable email notifications:
+
+1. Press *Tools & Settings*, then select *Notifications*.
+2. In the *SMTP Host* field, enter the SMTP host and port number.
+3. Select an *Authentication type*:
+   1. **Plain**&mdash;enter a username and password to be used for authentication.
+   2. **Login**&mdash;enter a username and password to be used for authentication.
+4. Enter the *From email address* and the *Display name*.
+5. Press *Verify* to ensure that the email configuration is working.
+6. Press *Save* when complete.
+
+For AI practitioners to receive notification emails for their jobs, they will need to add the configuration to the personal settings.
```

Comment:
Do we have another section for the AI practitioners? Maybe we should put it in a different page?